addons/compact-menu-bar/userstyle.css

```diff
@@ -0,0 +1,11 @@
+.account-nav_profile-name_2oRiV {
```

Comment:
We usually use `[class*=...]` for these style rules. In this case:
```suggestion
[class*="account-nav_profile-name"] {
```
<sub>[Learn more](https://github.com/ScratchAddons/ScratchAddons/pull/5595)</sub>